src/utils/index.ts

```diff
@@ -3,6 +3,13 @@ import storage from './storage'
 
 const FpZeroValue = new FPNumber(0)
 
+export const toHTTPS = (): void => {
+  const https = 'https:'
+  if (location.protocol !== https) {
```

Comment:
Could we place this code in `public/index.html` to `<head />` section, to made a redirect before loading resources (chunks)?